Can't import routeros_capsman_interface - HTTP 400 / Bad Request / unknown parameter configuration on PATCH

**Describe the bug**
When I import an existing RouterOS setup, `terraform apply` fails with HTTP 400 on routeros_capsman_interface refering a `configuration`.

This setup passes Terraform Plan, but fails Terraform Apply:

```
resource "routeros_capsman_interface" "master_downstairs_2g" {
  name    = "2ghz-MikroTik-2"
  comment = "2GHz downstairs"
  configuration = {
    config    = routeros_capsman_configuration.cfg_2ghz.name
  }
```

This setup fails Terraform Plan:
```
resource "routeros_capsman_interface" "master_downstairs_2g" {
  name    = "2ghz-MikroTik-2"
  comment = "2GHz downstairs"
  configuration = routeros_capsman_configuration.cfg_2ghz.name
```

Drilling more into Terraform state, I found that the resource is represented internally like this:

```
resource "routeros_capsman_interface" "master_downstairs_2g" {
  name    = "2ghz-MikroTik-2"
  comment = "2GHz downstairs"
  configuration = {
    "" = "my-2ghz-name"
  }
```
But it's not accepted by Terraform validation, either.

**To Reproduce**
1. set up CAP interface refering to a CAP configuration,
2. import into Terraform with `terraform import`
3. apply the setup,

**Expected behavior**
The setup should apply cleanly,

**Debug Information**
I'm attaching some debug information - I'll attach more if you need, I think this looks interesting:

```
2025-10-11T20:51:02.861+0200 [WARN]  Provider "registry.terraform.io/terraform-routeros/routeros" produced an invalid plan for routeros_capsman_interface.master_downstairs_2g, but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .mac_address: planned value cty.StringVal("48:8F:5A:86:72:1A") for a non-computed attribute
      - .configuration: planned value cty.MapVal(map[string]cty.Value{"":cty.StringVal("cfg-2ghz"), "config":cty.StringVal("cfg-2ghz")}) does not match config value cty.MapVal(map[string]cty.Value{"config":cty.StringVal("cfg-2ghz")}) nor prior value cty.MapVal(map[string]cty.Value{"":cty.StringVal("cfg-2ghz")})
      - .datapath: planned value cty.MapVal(map[string]cty.Value{"config":cty.StringVal("datapath1")}) for a non-computed attribute
      - .radio_mac: planned value cty.StringVal("48:8F:5A:86:72:1A") for a non-computed attribute
      - .radio_name: planned value cty.StringVal("488F5A86721A") for a non-computed attribute
      - .arp_timeout: planned value cty.StringVal("auto") for a non-computed attribute
      - .master_interface: planned value cty.StringVal("none") for a non-computed attribute
routeros_capsman_interface.master_downstairs_2g: Modifying... [id=*E]
2025-10-11T20:51:02.862+0200 [INFO]  Starting apply for routeros_capsman_interface.master_downstairs_2g
2025-10-11T20:51:02.863+0200 [DEBUG] routeros_capsman_interface.master_downstairs_2g: applying the planned Update change

(...)

2025-10-11T20:51:02.876+0200 [ERROR] provider.terraform-provider-routeros_v1.88.0.exe: Response contains error diagnostic: tf_provider_addr=terraform-routeros/routeros tf_req_id=fbfbef2c-596c-d1ad-9b03-c1da7e3beb7a tf_resource_type=routeros_capsman_interface @caller=github.com/hashicorp/terraform-plugin-go@v0.29.0/tfprotov5/internal/diag/diagnostics.go:58 @module=sdk.proto diagnostic_summary="PATCH 'https://192.168.2.1/rest/caps-man/interface/*E' returned response code: 400, message: 'Bad Request', details: 'unknown parameter configuration.'" tf_proto_version=5.10 tf_rpc=ApplyResourceChange diagnostic_detail="" diagnostic_severity=ERROR timestamp="2025-10-11T20:51:02.876+0200"
2025-10-11T20:51:02.877+0200 [DEBUG] State storage *cloud.State declined to persist a state snapshot
2025-10-11T20:51:02.877+0200 [ERROR] vertex "routeros_capsman_interface.master_downstairs_2g" error: PATCH 'https://192.168.2.1/rest/caps-man/interface/*E' returned response code: 400, message: 'Bad Request', details: 'unknown parameter configuration.'
2025-10-11T20:51:02.877+0200 [DEBUG] cloud/state: state read serial is: 100; serial is: 100
2025-10-11T20:51:02.877+0200 [DEBUG] cloud/state: state read lineage is: b60f7fec-b160-db4c-662e-98f8988d0a58; lineage is: b60f7fec-b160-db4c-662e-98f8988d0a58
╷
│ Error: PATCH 'https://192.168.2.1/rest/caps-man/interface/*E' returned response code: 400, message: 'Bad Request', details: 'unknown parameter configuration.'
│
│   with routeros_capsman_interface.master_downstairs_2g,
│   on config.tf line 323, in resource "routeros_capsman_interface" "master_downstairs_2g":
│  323: resource "routeros_capsman_interface" "master_downstairs_2g" {
│
╵
2025-10-11T20:51:04.055+0200 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2025-10-11T20:51:04.060+0200 [INFO]  provider: plugin process exited: plugin=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.88.0/windows_amd64/terraform-provider-routeros_v1.88.0.exe id=24444
2025-10-11T20:51:04.060+0200 [DEBUG] provider: plugin exited
```

**Stack Trace**
see above.

**Additional context**

RouterOS 7.19.6, 
hAP ac^3
Terraform provider 1.88.0

I'd appreciate any help - I wasn't able to find any resources on the Web that would help me fix this. 

Thanks!

--- a/routeros/resource_capsman_interface.go
+++ b/routeros/resource_capsman_interface.go
@@ -36,8 +36,8 @@ func ResourceCapsManInterface() *schema.Resource {
 		MetaId:           PropId(Id),
 		MetaSkipFields: PropSkipFields("current_authorized_clients", "current_basic_rate_set", "current_channel",
 			"current_rate_set", "current_registered_clients", "current_state"),
-		MetaTransformSet: PropTransformSet("channel.config: channel", "datapath.config: datapath",
-			"rates.config: rates", "security.config: security"),
+		MetaTransformSet: PropTransformSet("channel.config: channel", "configuration.config: configuration",
+			"datapath.config: datapath", "rates.config: rates", "security.config: security"),
 
 		KeyArpTimeout: PropArpTimeoutRw,
 		"bound": {
@@ -137,7 +137,7 @@ func ResourceCapsManInterface() *schema.Resource {
 		DeleteContext: DefaultDelete(resSchema),
 
 		Importer: &schema.ResourceImporter{
-			StateContext: schema.ImportStatePassthroughContext,
+			StateContext: ImportStateCustomContext(resSchema),
 		},
 
 		Schema: resSchema,

